### ISSUE TITLE
VS 2022 & Android 12 supported? 

### ISSUE THREAD

**Issue by rvalitov** [syncfusion/essential-ui-kit-for-xamarin.forms#87]
Please let me know if Visual Studio 2022 and Android 12 targeting are supported.

**Comment by PraveenGopalsamy** [comment]
Yes. Syncfusion Essential UIKit supports Visual Studio 2022 for Windows with Android 12. To learn more about UIKit Control, please check the following user guide documentation.

https://help.syncfusion.com/xamarin/uikit/gettingstarted#extension-for-windows
